### PR TITLE
Add Tic-Tac-Toe Game code in python

### LINKED ISSUE #14: Tic-Tac-Toe Game
# Description

Build a simple Tic-Tac-Toe game for multiplayer / single-player. [How the game works](https://en.wikipedia.org/wiki/Tic-tac-toe#Game_play).

## Language

- [ ] C++: @
- [ ] Java: @
- [x] Python: @arnab031 
- [ ] Other (Please specify)

## Type of issue

- [x] Feature
- [ ] Bug

The checklist will be updated by the maintainers/author of this issue according to the language in which snippets are added or are proposed to be added.

### PR DESCRIPTION
**Pull request by arnab031**
# Add Tic-Tac-Toe Game code in python

Fixes: #14 

## Type of change

- [Y] New feature (non-breaking change which adds functionality)

# Checklist:

- [Y] My code follows the style guidelines of this project
- [Y] I have performed a self-review of my own code
- [Y] I have commented my code, particularly in hard-to-understand areas
- [Y] I have made corresponding changes to the documentation
- [Y] My changes generates no new warnings
- [Y] I have added tests that prove my fix is effective or that my feature works
- [Y] New and existing unit tests pass locally with my changes
- [Y] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
+# Tic-Tac-Toe game for multiplayer
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]:
```suggestion
# Tic-Tac-Toe
```

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+## Tic-Tac-Toe
+Tic-Tac-Toe is a multiplayer simple game.
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]: This is redundant. If you want to add stuff about the working of the code you can do so under the `Description` sub-heading

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+## Code
+```python
+
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]:
```suggestion
```

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+from os import system, name
+
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]:
```suggestion
```

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+# import sleep to show output for some time period 
+from time import sleep
+
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md:11]:
```suggestion
```

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+
+# define our clear function
+def clear():
+    # for windows
+    if name == 'nt':
+        _ = system('cls')
+
+        # for mac and linux(here, os.name is 'posix')
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]:
```suggestion
    # for mac and linux(here, os.name is 'posix')
```

File: snippets/python/tic_tac_toe_game_using_python.md [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+    else:
+        _ = system('clear')
+
+
+clear()
+str1 = "."
```
Comment (sakshamarora1) [inline snippets/python/tic_tac_toe_game_using_python.md]: Can you come up with a better informative naming for these variables. For example, `box1`, `block1`, etc.
This goes for all the other variables too. To make it easier for other people to comprehend what's happening without wasting too much time!

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Good work! I have left some comments for changes, go through them :D

**Review by kaustubhgupta — APPROVED** [review]
LGTM :-)

**Review by sakshamarora1 — APPROVED** [review]

**Pull request merged**